infrastructure/docker/php-fpm/config/90-materiallist.ini

```diff
@@ -1,5 +0,0 @@
-; php.ini
-
-date.timezone = UTC
-expose_php = off
-memory_limit = 64M
```

Comment:
This limit seems important. It does not seem to be replicated in the new setup. Is that intentional?

Reply:
Yes and changed to default to 64M. The other vaules are default.